src/main/java/com/revature/domain/Batch.java

```diff
@@ -27,11 +27,10 @@
 	@Column(name="b_name")
 	private String name;
 
-	public Batch() {/*empty constructor needed*/}
+	public Batch() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)